easybuild/easyconfigs/g/GMT/GMT-5.4.3-foss-2018a.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'CMakeMake'
+
+name = 'GMT'
+version = '5.4.3'
+
+homepage = 'http://gmt.soest.hawaii.edu/'
+description = """GMT is an open source collection of about 80 command-line tools for manipulating
+ geographic and Cartesian data sets (including filtering, trend fitting, gridding, projecting,
+ etc.) and producing PostScript illustrations ranging from simple x-y plots via contour maps
+ to artificially illuminated surfaces and 3D perspective views; the GMT supplements add another
+ 40 more specialized and discipline-specific tools. """
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+# 'http://gmt.soest.hawaii.edu/files/download?name=' needs flash enabled browser
+source_urls = [
+    'ftp://ftp.soest.hawaii.edu/gmt',
+    'ftp://ftp.soest.hawaii.edu/gmt/legacy',
+    'ftp://ftp.soest.hawaii.edu/gshhg',
+    'ftp://ftp.soest.hawaii.edu/gshhg/legacy',
+    'ftp://ftp.soest.hawaii.edu/dcw',
+    'ftp://ftp.soest.hawaii.edu/dcw/legacy',
+]
+gshhg_ver = '2.3.7'
+dcw_ver = '1.1.3'
+sources = [
+    'gmt-%(version)s-src.tar.xz',
+    # coastlines, rivers, and political boundaries
+    'gshhg-gmt-%s.tar.gz' % gshhg_ver,
+    # country polygons
+    'dcw-gmt-%s.tar.gz' % dcw_ver,
+]
+patches = ['GMT-5.1.2_netCDF.patch']
+checksums = [
+    'ed00e380c3dc94a3aef4b7aeaaac0f3681df703dc614e8a15a1864e20b3fa2c8',  # gmt-5.4.3-src.tar.xz
+    '9bb1a956fca0718c083bef842e625797535a00ce81f175df08b042c2a92cfe7f',  # gshhg-gmt-2.3.7.tar.gz
+    '1395e772c3f2d2900c78260ad4a9df2fecd9216e362ad141762f7499bfeb4f23',  # dcw-gmt-1.1.3.tar.gz
+    '2ebe26d55521fba8d0eae48f662611491f7cc0e489833bded11628e9a71f252f',  # GMT-5.1.2_netCDF.patch
+]
+
+builddependencies = [('CMake', '3.10.2')]
+
+dependencies = [
+    ('PCRE', '8.41'),
+    ('GDAL', '2.2.3', '-Python-2.7.14'),
+    ('FFTW', '3.3.7'),
+    ('netCDF', '4.6.0'),
+    ('Ghostscript', '9.22', '-cairo-1.14.12'),
```

Comment:
@boegel why the change to `Ghostscript`?

Reply:
The `Ghostscript` we have with `foss/2018a` has this `versionsuffix` to keep it compatible with other stuff... Is that what you pointed out?